src/org/thoughtcrime/securesms/components/AudioView.java

```diff
@@ -223,6 +224,16 @@ public void onClick(View v) {
         Log.w(TAG, "playbutton onClick");
         if (audioSlidePlayer != null) {
           togglePlayToPause();
+          AudioManager am = (AudioManager)getContext().getSystemService(Context.AUDIO_SERVICE);
+          AudioManager.OnAudioFocusChangeListener focusChangeListener = new AudioManager.OnAudioFocusChangeListener() {
+              @Override
+              public void onAudioFocusChange(int focusChange) {
+                  if (focusChange == AudioManager.AUDIOFOCUS_LOSS) {
+                      pauseButton.performClick();
+                  }
+              }
+          };
+          am.requestAudioFocus(focusChangeListener, AudioManager.STREAM_MUSIC, AudioManager.AUDIOFOCUS_GAIN);
```

Comment:
ftr, `requestAudioFocus(listener, streamType, durarionHint)` is deprecated. instead, `requestAudioFocus(AudioFocussRequeset)` shall be used: https://developer.android.com/reference/android/media/AudioManager#requestAudioFocus(android.media.AudioManager.OnAudioFocusChangeListener,%20int,%20int)


however, i am fine with the old function, it may take a long time until it actually is deleted, and so we can shift this work to the future (and even save some if at the end the new replacement will be replaced by sth else :)